### DISCUSSION TITLE
This website can't be reached.

### DISCUSSION THREAD

**Discussion by purashy** [outline/outline#3989] [Self Hosting]
Hello,

I am using `v0.65.2` and everything goes well. Installation was successful. In the .env I changed the following values (ports):

```
URL=http://localhost:9151
PORT=9151
```

When I start the app using `yarn start`, I see the following:

`{"label":"lifecycle","level":"info","message":"Listening on http://localhost:9151"}`

So, here is where the issue starts. Navigating to `IP:9151` it basically shows that the website is not reachable.
Any help is appreciated!

<details>
  <summary>My .env file</summary>

    NODE_ENV=production
    SECRET_KEY=xxxx
    UTILS_SECRET=evenmorexxxx
    DATABASE_URL=postgres://username:password@localhost:5432/databasename
    DATABASE_URL_TEST=postgres://username:password@localhost:5432/outline-test
    DATABASE_CONNECTION_POOL_MIN=
    DATABASE_CONNECTION_POOL_MAX=
    REDIS_URL=redis://localhost:6379
    URL=http://localhost:9151
    PORT=9151
    COLLABORATION_URL=
    AWS_ACCESS_KEY_ID=get_a_key_from_aws
    AWS_SECRET_ACCESS_KEY=get_the_secret_of_above_key
    AWS_REGION=xx-xxxx-x
    AWS_S3_ACCELERATE_URL=
    AWS_S3_UPLOAD_BUCKET_URL=http://s3:4569
    AWS_S3_UPLOAD_BUCKET_NAME=bucket_name_here
    AWS_S3_UPLOAD_MAX_SIZE=26214400
    AWS_S3_FORCE_PATH_STYLE=true
    AWS_S3_ACL=privateSLACK_CLIENT_ID=xxxx
    SLACK_CLIENT_SECRET=xxxx
    GOOGLE_CLIENT_ID=
    GOOGLE_CLIENT_SECRET=
    AZURE_CLIENT_ID=
    AZURE_CLIENT_SECRET=
    AZURE_RESOURCE_APP_ID=
    OIDC_CLIENT_ID=
    OIDC_CLIENT_SECRET=
    OIDC_AUTH_URI=
    OIDC_TOKEN_URI=
    OIDC_USERINFO_URI=
    OIDC_USERNAME_CLAIM=preferred_username
    OIDC_DISPLAY_NAME=OpenID
    OIDC_SCOPES=openid profile email
    SSL_KEY=
    SSL_CERT=
    CDN_URL=
    FORCE_HTTPS=true
    ENABLE_UPDATES=true
    WEB_CONCURRENCY=1
    MAXIMUM_IMPORT_SIZE=5120000
    DEBUG=http
    SLACK_VERIFICATION_TOKEN=your_token
    SLACK_APP_ID=A0XXXXXXX
    SLACK_MESSAGE_ACTIONS=true
    GOOGLE_ANALYTICS_ID=
    SENTRY_DSN=
    SMTP_HOST=xxx
    SMTP_PORT=xxx
    SMTP_USERNAME=xxx
    SMTP_PASSWORD=xxx
    SMTP_FROM_EMAIL=xxx
    SMTP_REPLY_EMAIL=xxx
    SMTP_TLS_CIPHERS=xxx
    SMTP_SECURE=true
    DEFAULT_LANGUAGE=en_US
    RATE_LIMITER_ENABLED=true
    RATE_LIMITER_REQUESTS=1000
    RATE_LIMITER_DURATION_WINDOW=60
</details>

**Comment by vgwidt** [comment]
It sounds like you are running without Docker or a proxy, which I haven't done before.  Are you trying to access the webpage from the same server that it is running on?

**Reply by purashy** [reply]
Correct. I am selfhosting not via docker but on debian server. I am trying to access service via my main pc (same network). So i navigate to 192.168.x.x:9150

**Reply by vgwidt** [reply]
> Correct. I am selfhosting not via docker but on debian server. I am trying to access service via my main pc (same network). So i navigate to 192.168.x.x:9150

When you navigate to it from the client are you being redirected to HTTPS?  You have FORCE_HTTPS on in your .env but the server is entered as http.  If you change it be aware browser caching might result in redirects to continue.

I think you also will need to change localhost in URL in your .env file to a valid hostname or the IP address of the server if only accessing internally.  Check to make sure its listening properly using `ss -a`.  Also, if you have a firewall on your Debian server make sure to open the necessary ports.

**Reply by purashy** [reply]
You are a saint 👍 <3 Worked.
Had to disabled force_https as i have my own ssl via domain.
had to also change http://localhost:port to https://domain.com

However, now I get redirected to slack. which tells me to "allow" and I do so, I get the code via mail, i authorize using slack, but then i am right back at the login screen again. Not sure why. Its like a loop.

**Reply by purashy** [reply]
> However, now I get redirected to slack. which tells me to "allow" and I do so, I get the code via mail, i authorize using slack, but then i am right back at the login screen again. Not sure why. Its like a loop.

I see this error message:

```
{"error":"column \"updatedAt\" of relation \"events\" does not exist","stack":"UnauthorizedError: column \"updatedAt\" of relation
 \"events\" does not exist\n    at AuthenticationError (/root/outline/build/server/errors.js:35:34)\n    at accountProvisioner
 (/root/outline/build/server/commands/accountProvisioner.js:45:43)\n    at processTicksAndRejections
 (node:internal/process/task_queues:96:5)\n    at async Strategy._verify
 (/root/outline/build/server/routes/auth/providers/slack.js:56:22)","level":"error","message":"Error during authentication"}
```

**Reply by vgwidt** [reply]
Not too sure on this one.. It might be related to https://github.com/outline/outline/commit/24170e868494fb0880401553bac5e83cc8896562

Maybe running yarn sequelize:migrate will help.
